docs/legacy-filtering.md

```diff
@@ -3,7 +3,7 @@ This page describes the old style of filtering and is deprecated and removed in
 agent version 5.0+. See [Filtering](filtering.md) for how to configure
 filtering in SignalFx Smart Agent 4.7.0+.
 
-## Old-style whitelist filtering
+## Old-style allow list filtering
```

Comment:
Same thing as my other comment, we use `metricsToExclude` regarding legacy filtering, so `Inclusion list` I think would read better